majifix311/src/test/java/com/example/majifix311/DatabaseHelperTest.java

```diff
@@ -89,7 +89,7 @@ public void accept(ArrayList<Problem> problems) throws Exception {
         //        serverResponses, onProblemsSavedInDatabase(), onError(), false);
 
         assertNotNull(mProblemResult);
-        ProblemTest.assertGetMatchesMock(mProblemResult.get(0));
+        ProblemTest.assertPostMatchesMock(mProblemResult.get(0));
```

Comment:
So, I see one problem with this: it's not testing all the fields we're putting in the db (ticket number, status, dates, attachments), and I really would like it to. Can you change this test to do so?